docker-compose.yml

```diff
@@ -306,7 +306,7 @@ services:
       - ./config/sparql-authorization-wrapper:/config
     restart: always
     logging: *default-logging
-  vendor-data-distribution:
+  vendor-data:
```

Comment:
Nitpick; but I rather keep the 'distribution' in the name. Vendor-data sound more like a separate db.

Reply:
Yeah I used `vendor-data` and Riad used `vendor-data-distribution`. During merge conflicts, I had to choose and I chose the shortest name because I don't like typing while debugging. Changed it back for clarity.